### ISSUE TITLE
No response for tapping FAQ the 2nd time 

### ISSUE THREAD

**Issue by DeXtmL** [shadowsocks/shadowsocks-android#1907]
1. open main config window
2. open side window
3. click FAQ, a browser window opens
4. click back to return to the config window
5. open side window
6. click FAQ, observe that nothing happens

**Expected behavior**
FAQ webpage correctly opens.

**Workaround**
quit to android home screen then reopen the config window then click FAQ.

Tested on
Pixel / Pixel 2 XL / Galaxy S7 / Galaxy S8

**Comment by madeye** [comment]
What's the version of your shadowsocks?

I cannot reproduce it with latest version of shadowsocks on Pixel phones.

**Comment by DeXtmL** [comment]
The latest version from play store 4.5.7.
To reproduce it, the key is to remain in the app when closing the FAQ window (to me it's an in-app browser window), by either click back or tap the close button on top left. After closing, tap FAQ button again to see it.

**Comment by DeXtmL** [comment]
Same issue with version 4.6.1.

**Comment by iKirby** [comment]
I can reproduce this on both Pixel (Android 9) and Nexus 9 (Android 7.1.2) using version 4.6.1.

**Comment by madeye** [comment]
@Mygod could you reproduce it on your device?

**Comment by Mygod** [comment]
Not on the latest commit. Haven't tried 4.6.1 but I don't think I have ever encountered this.

**Comment by Mygod** [comment]
Which browser are you using?

**Comment by studentmain** [comment]
Shadowsocks 4.6.0 reproduced with Android 8 and Chrome 68

```
Google Chrome 68.0.3440.85 (正式版本) （32 位）
修订版本 ad3f46192cdf21bdd00f8355cc139b7ca903ba17-refs/branch-heads/3440@{#775}
操作系统 Android 8.0.0; Mi A1 Build/OPR1.170623.026
Google Play 服务 SDK=11200000; Installed=12874025; Access=1p
JavaScript V8 6.8.275.24
```

Mygod <notifications@github.com> 于 2018年8月8日周三 14:38写道：

**Comment by Mygod** [comment]
This should be fixed in 4.6.2.
